code/randomized-algorithms/shuffle_an_array/Shuffle_An_Array.java

```diff
@@ -0,0 +1,28 @@
+// Part of Cosmos by OpenGenus Foundation 
+
+import java.util.Random;
+import java.util.Arrays;
+
+public class Shuffle_An_Array
+{
+  static void randomizing(int array[], int n)
+	{
+	       Random r  = new Random();
+	       for(int i = n-1;i>0;i--)
+            {
+              int j = r.nextInt(i);
+              int temp = array[i];
+              array[i] = array[j];
+              array[j] = temp;
+            }
+		System.out.println(Arrays.toString(array));
+	}
+  
+   public static void main(String[]args)
+  {
+          int[] A = {1,2,3,4,5,6,7,8};
+          int n = A.length();
```

Comment:
length is a property of an array in `Java` but you are referring to an undefined function. You, probably, want this statement: ` int n = A.length;`

Reply:
I fixed it 👍

Reply:
Thank you ! 👍